src/Diagnostics.RuntimeHost/Controllers/DevOpsController.cs

```diff
@@ -76,14 +76,29 @@ public async Task<IActionResult> PushChangesAsync([FromBody]JToken jsonBody)
             string resourceUri = jsonBody[$"resourceUri"].ToString();
 
             object response = await _devOpsClient.PushChangesAsync(branch, files, repoPaths, comment, changeType, resourceUri, this.HttpContext.Request.Headers[RequestIdHeaderName]);
-            return Ok(response);
+            if (response.GetType() != typeof(BadRequestObjectResult))
```

Comment:
Can you always return `(IActionResult)response` ? Will that work in success scenarios as well?

Reply:
I just tried this out. Trying to cast the success response to IActionResult throws an InvalidCastException. It does still successfully push but UI will get a 500 instead of the success response

Reply:
With new change of handling existing active PR, we don't need to check for `BadRequestObjectResult`, correct?

Reply:
This is in the push. The PR doesnt check for bad request. Push creates a BadRequest when the ID is already in use because duplicate IDs try to create a file that already exists in the repo and that causes failure. the other bad request check i added accidentally. im about to remove that one